crates/sui-bridge-indexer/src/config.rs

```diff
@@ -3,14 +3,14 @@
 
 use anyhow::Result;
 use serde::Deserialize;
-use std::{fs, path::Path};
+use std::{env, fs, path::Path};
 
 /// config as loaded from `config.yaml`.
 #[derive(Debug, Clone, Deserialize)]
 pub struct Config {
     pub remote_store_url: String,
     pub eth_rpc_url: String,
-    pub db_url: String,
+    pub db_url: Option<String>,
```

Comment:
should we just get rid of this in the config and rely on env var

Reply:
This would be easier